db/app.js

```diff
@@ -10,10 +14,12 @@ app.get("/api", (req, res) => {
 
 app.get("/api/topics", getTopics);
 
-app.get("/api/articles/:article_id", getArticle);
+app.get("/api/articles/:article_id", getArticles);
 
-app.use("/*", (req, res, next) => {
-  res.status(400).send({ msg: "bad request" });
+app.get("/api/articles", getWholeArticle);
```

Comment:
your endpoint is to get all of the articles, but your controller is called `getWholeArticle`. Your controller name suggests you're getting one article, your endpoint is designed to get them all. Something to think about!